src/main/java/com/zendesk/maxwell/MaxwellConfig.java

```diff
@@ -23,6 +23,12 @@
 	public Integer mysqlPort;
 	public String  mysqlUser;
 	public String  mysqlPassword;
+
+	public String mysqlSchemaHost;
+	public Integer mysqlSchemaPort;
```

Comment:
I think shoudl make a MySQLConfig class and have two instances of the new class rather than the repeated code you see here.